### PR TITLE
Clean up FileHandler and FileHandlerTest

### PR DESCRIPTION
**Pull request by Akcbryant**
-Dry up deciding which type of filehandler to choose
-Use constants from FilHandler in the tests
-Extract strings into constants

### CODE REVIEW

**Comment by Akcbryant** [comment]
Tony, I originally had the file handler decision in the request handler and I think you are right that is the best option.

File: src/main/java/javaserver/handlers/FileHandler.java [hunk 1]
```diff
@@ -57,8 +51,14 @@ public Response handleRequest(Request request) {
         }
     }
 
+    private Handler getTypeOfFileHandler(Request request) {
+        if (isPartialRequest(request)) return new PartialHandler(fileUri, resourceUtility);
+        if (isImageRequest(request)) return new ImageHandler(fileUri, resourceUtility);
```
Comment (tbaik) [inline src/main/java/javaserver/handlers/FileHandler.java]: https://www.imperialviolet.org/2014/02/22/applebug.html

:smile: 

TL;DR: use {}s

File: src/main/java/javaserver/handlers/FileHandler.java [hunk 2]
```diff
@@ -23,24 +24,17 @@ public FileHandler(String fileUri, ResourceUtility resourceUtility) {
...
 
-        if (isPartialRequest(request)) {
-            response = new PartialHandler(fileUri, resourceUtility).handleRequest(request);
-            return response;
-        }
-
-        if (isImageRequest(request)) {
-            response = new ImageHandler(fileUri, resourceUtility).handleRequest(request);
-            return response;
+        Handler fileHandler = getTypeOfFileHandler(request);
```
Comment (tbaik) [inline src/main/java/javaserver/handlers/FileHandler.java:32]: I meant that another class should have the responsibility of doing this. Maybe a FileHandlerDeterminer? I know this class is still relatively small but in real world cases, it would be more maintainable that way.

Comment (Akcbryant) [inline src/main/java/javaserver/handlers/FileHandler.java:32]: Oh i see, so still within the FileHandler. Ok I'll think about those two options and see which one makes the most sense.
